dts/bindings/sensor/st,lis2du12-common.yaml

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) 2023 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    When setting the accel-range, accel-odr, properties in a .dts or .dtsi
+    file you may include lis2du12.h and use the macros defined there.
+
+    Example:
+    #include <zephyr/dt-bindings/sensor/lis2du12.h>
+
+    lis2du12: lis2du12@0 {
+      ...
+
+      accel-range = <LSM6DSV16X_DT_FS_8G>;
+      accel-odr = <LSM6DSV16X_DT_ODR_AT_60Hz>;
+    };
+
+include: sensor-device.yaml
+
+properties:
+  int1-gpios:
+    type: phandle-array
+    description: |
+      INT1 pin
+
+      This pin defaults to active high when produced by the sensor.
+      The property value should ensure the flags properly describe
+      the signal that is presented to the driver.
+
+  int2-gpios:
+    type: phandle-array
+    description: |
+      INT2 pin
+
+      This pin defaults to active high when produced by the sensor.
+      The property value should ensure the flags properly describe
+      the signal that is presented to the driver.
+
+  drdy-pin:
+    type: int
+    default: 1
+    description: |
+      Select DRDY pin number (1 or 2).
+
+      1 = drdy is generated from INT1
+      2 = drdy is generated from INT2
+
+      This number represents which of the two interrupt pins
+      (INT1 or INT2) the drdy line is attached to. This property is not
+      mandatory and if not present it defaults to 1 which is the
+      configuration at power-up.
+    enum: [1, 2]
+
+  accel-range:
+    type: int
+    default: 0
+    description: |
+      Range in g. Default is power-up configuration.
```

Comment:
nit: it would be great to consider doing a mass replace of s/g/G/ as there are plenty of places where the unit is mispelled (applies to "mg" just below too). It concerns mostly the ST accelerometers but a few others as well.

Reply:
> nit: it would be great to consider doing a mass replace of s/g/G/ as there are plenty of places where the unit is mispelled (applies to "mg" just below too). It concerns mostly the ST accelerometers but a few others as well.

@kartben in all out datasheet I read g/mg instead of G/mG. I'm failing to find with google a clear advise on which one to use...

Reply:
Mmmm yeah my bad. It would seem that "g" is indeed the way to go. I guess people preferred risking a confusion with "grams" than a confusion with G the gravitational constant. It hurts my brain, but definitely not something that needs fixing in the end, sorry for the noise!